<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <widget class="QLineEdit" name="userNameTextField">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextField">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>You dont have an account?</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login Here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redirectButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Register Here </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Forgot Password?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redirectResetButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Click Here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
